terraform/variables.tf

```diff
@@ -79,6 +79,10 @@ variable "private_subnet_id" {
   description = "Identifier for private AWS subnet"
 }
 
+variable "peers" {
+  description = "List of initial Supervisor peers"
+}
```

Comment:
Actually I should change this to a list and not just a string. It *must* be a string to pass to the template resource but it doesn't need to be when we receive it at this point